src/App/Pages/Accounts/LoginPasswordlessRequestViewModel.cs

```diff
@@ -255,6 +261,15 @@ private async Task CreatePasswordlessLoginAsync()
             var response = await _authService.PasswordlessCreateLoginRequestAsync(_email, AuthRequestType);
             if (response != null)
             {
+                //TODO TDE if is admin type save to memory to later see if it was approved
+                /*
+                  const adminAuthReqStorable = new AdminAuthRequestStorable({
+                      id: reqResponse.id,
+                      privateKey: this.authRequestKeyPair.privateKey,
+                    });
+
+                    await this.stateService.setAdminAuthRequest(adminAuthReqStorable);
+                */
```

Comment:
❓ Will this be addressed in a different PR? If so, could we add a reference to the task id here if such task exists?

Reply:
it's already addressed in this pr: https://github.com/bitwarden/mobile/pull/2643